### ISSUE TITLE
[1.6b23][1.5.2b11] Including a cordova plugin via relative path breaks rebuilds

### ISSUE THREAD

**Issue by BudgieInWA** [meteor/meteor#9033]
In the current beta versions of both 1.5.2 and 1.6, including a plugin via a relative path causes rebuilds to fail.

If `cordova-plugins` contains `local-plugin@file://../plugin` for example, the first build will succeed, but subsequent builds with fail with:

```
=> Errors executing Cordova commands:         
											  
   While removing plugins cordova-plugin-meteor-webapp,cordova-plugin-splashscreen,cordova-plugin-statusbar,cordova-plugin-whitelist,cordova-plugin-wkwebview-engine,local-plugin from Cordova project:
   Cordova error: Plugin "local-plugin" is not present in the project. See `cordova plugin list`.
   (If the error message contains suggestions for a fix, note that this may not apply to the Meteor integration. You can try running again with the --verbose option to help diagnose the issue.)
```

This error is reproduced in a bare-bones project on the `local-plugins-break-build` branch of this repo: https://github.com/BudgieInWA/meteor-cordova-postBuildExtras-bug/tree/local-plugins-break-build

**Comment by whileoneloop** [comment]
It looks like this has been fixed in cordova-lib master.  Here was the pull request:

https://github.com/apache/cordova-lib/pull/557

... which was merged into cordova-lib master on 3-Jun.  Cordova's 7.0.1 release was 10-May-17, it doesn't include the 3-Jun patch to resolve this issue.  Perhaps the next 7.0.x release will include this fix.

**Comment by skirunman** [comment]
This is fixed in[ cordova-lib@7.1.0](https://github.com/apache/cordova-lib/tree/rel/7.1.0) which is merged in [Meteor 1.6.1](https://github.com/meteor/meteor/pull/9213)

**Comment by hwillson** [comment]
As @skirunman mentioned, this is fixed in 1.6.1 and will be coming soon. Closing - thanks!
